docs/modules/user-guide/partials/proc_devfile-quick-reference.adoc

```diff
@@ -0,0 +1,263 @@
+[id="proc_devfile-quick-reference_{context}"]
+= Devfile quick reference
+
+[role="_abstract"]
+This page quickly introduces you to devfile properties. The code snippets help you get started developing your own devfile projects. The tables show you what the different devfile properties require if you use these properties. Refer to the table of contents to go to the devfile property you want to work with.
+
+== SchemaVersion
+
+[source,yaml]
+----
+v2.0
+---
+schemaVersion: 2.0.0
+----
+
+[cols="1,1"]
+|===
+|Key
+|schemaVersion
+
+|Type
+|string
```

Comment:
Type String is capitalized in some languages, e.g. in Java.
Sorry, I lack context, maybe it's not important here.

Reply:
Good point. I'll leave as is (un-capitalized) because it's not capitalized on the odo page where we are taking this content.